lib/chef_backup/helpers.rb

```diff
@@ -194,9 +201,10 @@ def reconfigure_marketplace
   end
 
   def enabled_addons
-    SERVER_ADD_ONS.select do |_name, config|
+    SERVER_ADD_ONS.select do |name, config|
       begin
-        File.directory?(File.dirname(config['config_file']))
+        File.directory?(addon_install_dir(name)) &&
+          File.directory?(File.dirname(config['config_file']))
       rescue
```

Comment:
Just curious, which failure does this try to fix? It seems to me that it's not the `File` methods, since they're quite robust:

```
irb(main):001:0> File.directory?("ivemadethisup")
=> false
irb(main):003:0> File.dirname("ivemadethisup")
=> "."
```

and [`addon_install_dir`](https://github.com/chef/chef_backup/pull/25/files#diff-e36ba4efd58fce92894f699654432675R121) doesn't throw anything, either.

So, if this is only about `config['config_file']`, why not just add
something like `return false unless config['config_file']`?